guides/05_tabular-data-science-and-plots/creating-a-dashboard-from-supabase-data.md

```diff
@@ -0,0 +1,122 @@
+# Creating a Dashboard from Supabase Data
+
+Tags: TABULAR, DASHBOARD, PLOTS 
+
+Supabase is a cloud-based open-source backend that provides a PostgreSQL database, authentication, and other useful features for building web and mobile applications. In this tutorial, we will show you how to read data from Supabase plot it in **real-time** on a Gradio Dashboard.
```

Comment:
maybe add a link to supabase.com? 

typo:
> data from Supabase plot it in

data from Supabase *and plot it in